joeynmt/embeddings.py

```diff
@@ -53,3 +61,39 @@ def forward(self, x: Tensor) -> Tensor:
     def __repr__(self):
         return "%s(embedding_dim=%d, vocab_size=%d)" % (
             self.__class__.__name__, self.embedding_dim, self.vocab_size)
+
+    #from fairseq
+    def load_from_file(self, embed_path: str, vocab: Vocabulary):
+        """Load pretrained embedding weights from text file.
+
+        - first line will be ignored (vocabulary size and dimension)
```

Comment:
Maybe clarify: First line is expected to contain vocabulary size and dimension. The dimension has to match the model's specified embedding size, the vocabulary size is not used.

Reply:
It would also be great to add a line about what happens with vocabulary items that are not part of the pretrained vocabulary.